StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkLoginWarmup/NetworkingLinkLoginWarmupBodyView.swift

```diff
@@ -69,7 +69,7 @@ private func CreateAvatarView(email: String) -> UIView {
 
     let letterLabel = AttributedLabel(
         font: .body(.small),
-        textColor: .legacyLinkBrand600
+        textColor: .linkGreen900
```

Comment:
We now use the new Link colors here, no matter the flow. 

| Financial Connections | Instant Debits |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 - 2024-07-23 at 14 25 55](https://github.com/user-attachments/assets/d6a97bef-12b1-4285-88e3-c27a39812703) | ![Simulator Screenshot - iPhone 15 - 2024-07-23 at 14 03 23](https://github.com/user-attachments/assets/e0162e97-c079-42c7-b5cd-b947699a6b62) |

Reply:
yea that makes a lot of sense to just use the colors directly instead of adding extra abstractions 💯